interface/new/new_comprehensive.php

```diff
@@ -380,61 +414,73 @@ function validate(f) {
           return false;
         }
         <?php } ?>
-        var msg = "";
         var ins_error = '0';
-        msg += "<?php echo htmlspecialchars(xl('The following fields are required'),ENT_QUOTES); ?>:\n\n";
         for ( var i = 0; i < errMsgs.length; i++ ) {
-               msg += errMsgs[i] + "\n";
+          iziToast.error({
+            title: 'Error',
+            message: "<?php echo htmlspecialchars(xl('The following field is required: '),ENT_QUOTES); ?>" + errMsgs[i],
+            position: 'topRight',
+          })
         }
         <?php if($GLOBALS['primary_insurance_required']){ ?>
           if($('#i1provider option:selected').val() == '') {
-             var ins_error = '1';
-             msg += "<?php echo htmlspecialchars(xl('Primary Insurance Provider Entry Required.'),ENT_QUOTES); ?> \n";
+            var ins_error = '1';
```

Comment:
The "required" list...are all of these configured somewhere?  The Layout engine has a tool for setting layout elements as required or not (only works on some, probably means they are handled separately).  Being able to control "required" fields for creating new patients and the like (and for duplicate checking) is darn important...and that is a feature we don't have so good.  State/region is one of those items that winds up creating a lot of screwups if it is not entered.
-So, when I test this, Do I just need to create a patient and hit various fields, right?  What if I change some demographics fields in the layout options?